akka-stream-tests/src/test/resources/reference.conf

```diff
@@ -5,6 +5,7 @@ akka {
     serialize-messages = on
     default-dispatcher.throughput = 1 // Amplify the effects of fuzzing
   }
+  akka.actor.warn-about-java-serializer-usage = false
```

Comment:
I guess that is because of String and int messages. Perhaps we should add `java.lang` to our ignored prefixes?